File: ReactNativeDemo/app/src/main/java/com/mjiayou/reactnativedemo/MainActivity.java
package com.mjiayou.reactnativedemo;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.tvHello).setOnLongClickListener(new View.OnLongClickListener() {
            @Override
            public boolean onLongClick(View v) {
                startActivity(new Intent(MainActivity.this, MyReactActivity.class));
                return false;
            }
        });
    }
}

File: HelloKotlin/app/src/main/java/com/mjiayou/hellokotlin/MainActivity.kt
package com.mjiayou.hellokotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.mjiayou.hellokotlin.databinding.ActivityMainBinding
import com.mjiayou.hellokotlin.view.MovieFragment

class MainActivity : AppCompatActivity() {

    lateinit var mBinding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_main) // 通过 DataBindingUtil 获取 ActivityMainBinding

        supportFragmentManager.beginTransaction().add(R.id.fl_container, MovieFragment.getInstance()).commit()
    }
}